src/test/components/TimelineMarkers.test.js

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import React from 'react';
+import TimelineMarkers from '../../content/containers/TimelineMarkers';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import mockCanvasContext from '../fixtures/mocks/canvas-context';
+import { storeWithProfile } from '../fixtures/stores';
+import { getProfileWithMarkers } from '../store/fixtures/profiles';
+
+jest.useFakeTimers();
```

Comment:
I don't really like that we call that here at the top level, I'd rather call it in `beforeEach` so that we can call `useRealTimers` in an `afterEach`.

That said I don't really understand why we use fake timers -- don't you need real timers to mock `requestAnimationFrame` ?